doc/guides/running_simulations.rst

```diff
@@ -199,3 +199,45 @@ a Poisson spike train using different seeds and output files for each run:
         nest.Connect(nrn, sd)
     
         nest.Simulate(100)
+
+Monitoring elapsed time
+-----------------------
+
+The progress of the simulation can be monitored by setting:
+
+::
+
+    SetKernelStatus({"print_time": True})
+
+If enabled, a line is printed to screen at every time step of the simulation to
+track the percentage and the absolute elapsed biological simulation time as
+well as the real-time factor, for example:
```

Comment:
I would add a comma to list these three items, i.e. ``to track the percentage, the absolute elapsed biological simulation time and the real-time factor, for example:``

Reply:
Initially, I intended "percentage" to be an adjective and therefore chose the previous formulation. But your suggestion is probably easier to understand and I changed accordingly.